binaries/cli/src/main.rs

```diff
@@ -90,7 +90,10 @@ enum Command {
     // Planned for future releases:
     // Dashboard,
     /// Show logs of a given dataflow and node.
-    Logs { dataflow: String, node: String },
+    Logs {
+        dataflow_or_node: String,
+        node: Option<String>,
+    },
```

Comment:
I found these argument names confusing, so I looked through the `clap` docs to see whether there is a way to make the first argument optional. I found the [`allow_missing_positional`](https://docs.rs/clap/latest/clap/struct.Command.html#method.allow_missing_positional) flag, which looks like it would be useful here:

```suggestion
    #[command(allow_missing_positional = true)]
    Logs {
        dataflow: Option<String>,
        node: String,
    },
```

This way, the help text becomes easier to understand too:

> Usage: dora logs [DATAFLOW] <NODE>

Reply:
Thanks for looking for it!